### DISCUSSION TITLE
How to change boot logo for Jetson Orin NX

### DISCUSSION THREAD

**Discussion by NavSriv** [OE4T/meta-tegra#1779] [Q&A]
I am using [Diamond Systems Jetson Orin NX 16GB](https://www.diamondsystems.com/products/jackson) board for development. I am building image for p3768-0000-p3767-0000 using kirkstone branch. I would like to change boot logo from Nvidia logo to custom logo, though I am not sure the one I am seeing on the board is coming from bootloader or kernel framebuffer. How can I achience this?

**Comment by dwalkes** [comment]
I believe as of Jetpack 5 (see [release notes](https://github.com/OE4T/meta-tegra/wiki/JetPack-5.0.2-L4T-R35.1.0-Notes)) boot logos come from the UEFI bootloader.  I don't think we have documented instructions to replace but the instructions in https://github.com/NVIDIA/edk2-nvidia/issues/100 might be helpful. If you could include some detail or an example project which shows what you did I'm sure others would find it useful.

**Reply by NavSriv** [reply]
As of now I haven't done anything except giving support for board to come up. I am wondering what changes to make in order to change this logo appearing during start of the board initialization.
